maps/random_ruins/exoplanets/konyang/village.dm

```diff
@@ -0,0 +1,55 @@
+/datum/map_template/ruin/exoplanet/konyang_village
+	name = "Konyang Village"
+	id = "konyang_village"
+	description = "A rural village on Konyang."
+
+	template_flags = TEMPLATE_FLAG_NO_RUINS|TEMPLATE_FLAG_RUIN_STARTS_DISALLOWED
+	sectors = list(SECTOR_HANEUNIM)
+	suffixes = list("konyang/village.dmm")
+
+/area/konyang_village
+	name = "Konyang Village"
+	icon_state = "bluenew"
+	requires_power = FALSE
+	dynamic_lighting = TRUE
+	no_light_control = FALSE
+	base_turf = /turf/simulated/mineral
+	flags = HIDE_FROM_HOLOMAP
+
+/datum/ghostspawner/human/konyang_village
+	short_name = "konyang_village"
+	name = "Konyang Villager"
```

Comment:
Shouldn't this be consistently called "Konyang Village**r**", in short name, name and path?